src/status_im2/contexts/chat/lightbox/style.cljs

```diff
@@ -77,8 +78,24 @@
    :align-items        :center
    :justify-content    :center})
 
-(def text-style
-  {:color             colors/white
-   :align-self        :center
-   :margin-horizontal 20
-   :margin-vertical   12})
+
+(defn background
+  [{:keys [overlay-opacity]} {:keys [overlay-z-index]}]
+  (reanimated/apply-animations-to-style
+   {:opacity overlay-opacity}
+   {:background-color colors/neutral-100-opa-70
+    :position         :absolute
+    :top              0
+    :bottom           0
+    :z-index          @overlay-z-index
+    :left             0
+    :right            0}))
+
+;; This is needed so that text does not show in the bottom inset part as it is transparent
```

Comment:
I think this comment would be easier to comprehend if you moved it to where the function is used. I had to inspect the hiccup in `bottom-view` to figure out.